standards/SPIFFE_Workload_API.md

```diff
@@ -360,6 +366,122 @@ When interfacing with legacy JWT validators, the `FetchJWTBundles` method can be
+  // Bundles are encoded in JWK set (RFC 7517) format.
+  map<string, string> bundles = 1;
+}
+```
+
+### 7.2 Profile RPCs
+
+#### 7.2.1 FetchWITSVID
+
+The `FetchWITSVID` RPC streams back WIT-SVIDs. 
+
+The `WITSVIDRequest` request message is currently empty and is a placeholder for future expansion.
+
+The `WITSVIDResponse` response consists of a mandatory `svids` field, which MUST contain one or more `WITSVID` messages (one for each identity granted to the client).
+
+All fields in the `WITSVID` message are mandatory, with the exception of the `hint` field. When the `hint` field is set (i.e. non-empty), SPIFFE Workload API servers MUST ensure its value is unique amongst the set of returned SVIDs in any given `WITSVIDResponse` message. In the event that a SPIFFE Workload API client encounters more than one `WITSVID` message with the same `hint` value set, then the first message in the list SHOULD be selected.
+
+If the client is not entitled to receive any WIT-SVIDs, then the server SHOULD respond with the "PermissionDenied" gRPC status code (see the [Error Codes](SPIFFE_Workload_Endpoint.md#6-error-codes) section in the SPIFFE Workload Endpoint specification for more information). Under such a case, the client MAY attempt to reconnect with another call to the `FetchWITSVID` RPC after a backoff.
+
+As mentioned in [Stream Responses](#43-stream-responses), each `WITSVIDResponse` message returned on the `FetchWITSVID` stream contains the complete set of authorized SVIDs for the client at that point in time. As such, if the server redacts SVIDs from a subsequent response (or all SVIDs, i.e., returns a "PermissionDenied" gRPC status code) the client SHOULD cease using the redacted SVIDS.
+
+#### 7.2.2 FetchWITBundles
+
+The `FetchWITBundles` RPC streams back bundles for both the trust domain in which the server resides and foreign trust domains. These bundles MUST only be used to authenticate WIT-SVIDs.
+
+The `WITBundlesRequest` request message is currently empty and is a placeholder for future expansion.
+
+The `WITBundlesResponse` response message consists of a mandatory `bundles` field, which MUST contain at least the trust bundle for the trust domain in which the server resides.
+
+If the client is not entitled to receive any WIT bundles, then the server SHOULD respond with the "PermissionDenied" gRPC status code (see the [Error Codes](SPIFFE_Workload_Endpoint.md#6-error-codes) section in the SPIFFE Workload Endpoint specification for more information). The client MAY attempt to reconnect with another call to the `FetchWITBundles` RPC after a backoff.
+
+As mentioned in [Stream Responses](#43-stream-responses), each `WITBundleResponse` response contains the complete set of authorized trust bundles for the client at that point in time. As such, if the server redacts bundles from a subsequent response (or all bundles, i.e., returns a "PermissionDenied" gRPC status code) the client SHOULD cease using the redacted bundles.
```

Comment:
Should we support some kind of pagination API here (and in the other profiles)? gRPC has some default receive message sizes (I think it's 4MB). If you have large bundles or large number of federation relationships, you may end up not being able to receive the updates anymore.

Reply:
Hm - it's a good point. We should discuss this in next SIG-Spec since I think I think modifying the existing profiles will merit further discussion esp in regards to back/fore compatibility.

Reply:
I have included a proposed design for pagination in [713f75e](https://github.com/spiffe/spiffe/pull/327/commits/713f75e3465d545b99003fbd7c24f71d57f64685).